CSS/Utilities/Text-Utility.css

```diff
@@ -0,0 +1,46 @@
+.text-utility-container {
+  width: 80%;
+  padding: var(--sp-md);
+  box-shadow: 0 0 4px var(--gray-900);
+}
+.text-utility-container > * {
+  margin-bottom: var(--sp-sm);
+}
```

Comment:
Here, are you trying to give margin-bottom to all the children under this class or the class itself?

Reply:
To all the children's within text utility container.

Reply:
Will that be fine or would you like to suggest any good approach